packages/medusa-fulfillment-webshipper/src/services/webshipper-fulfillment.js

```diff
@@ -699,6 +699,13 @@ class WebshipperFulfillmentService extends AbstractFulfillmentService {
       webShipperItem.tarif_number = tarifNumber
     }
 
+    const weight = item?.variant?.weight || item?.variant?.product?.weight
+
+    if (weight) {
```

Comment:
I guess it doesn't make a lot of sense for something to have 0 weight and to be shippable, but it's better of we check undefined here. We just need to make sure the weights are not set to 0 by default, so undefined will never happen

Reply:
yeah maybe its ok to allow 0 values to be sent.